include/fmt/core.h

```diff
@@ -1334,21 +1334,12 @@ template <typename Char>
 struct is_contiguous<internal::basic_buffer<Char> >: std::true_type {};
 
 /** Formats a string and writes the output to ``out``. */
-template <typename Container>
-typename std::enable_if<
-  is_contiguous<Container>::value, std::back_insert_iterator<Container>>::type
-    vformat_to(std::back_insert_iterator<Container> out,
-               string_view format_str, format_args args) {
-  internal::container_buffer<Container> buf(internal::get_container(out));
-  vformat_to(buf, format_str, args);
-  return out;
-}
-
-template <typename Container>
+template <typename Container, typename Char>
 typename std::enable_if<
   is_contiguous<Container>::value, std::back_insert_iterator<Container>>::type
     vformat_to(std::back_insert_iterator<Container> out,
-               wstring_view format_str, wformat_args args) {
+               basic_string_view<Char> format_str,
```

Comment:
Will this work on older gcc if `format_str` is a string literal? Could you add a test?

Reply:
Is this particular overload of `vformat_to` with non-variadic parameters meant to be a public interface? AFAICS it is called only from other interface functions which do have a variadic interface and explicitly convert the given `format_string` to a `basic_string_view`. I am asking because I have a hard time to come up with a test to exercise this `vformat_to` overload.

Reply:
Nevermind, I've widened the interface and added some tests.

Reply:
Whoa, the widened but contrained interface triggers a spectacular compile failure on all versions of gcc and clang. I suspect a weird interaction with their stl implementation. MSVC is happily chugging along, though ...

Reply:
For the sake of documentation: [Travis log](https://travis-ci.org/fmtlib/fmt/builds/436191568)
Removing the constraint now ...

Reply:
No matter what I do, gcc 4.8 is choking. I'd rather stick with my original code!